br/pkg/streamhelper/advancer_env.go

```diff
@@ -32,6 +37,13 @@ type PDRegionScanner struct {
 	pd.Client
 }
 
+// Updates the service GC safe point for the cluster.
+// Returns the latest service GC safe point.
```

Comment:
it seems UpdateServiceGCSafePoint return the minimum service GC safepoint. not the latest?